pkg/services/accesscontrol/seeder/roles.go

```diff
@@ -0,0 +1,68 @@
+package seeder
+
+import "github.com/grafana/grafana/pkg/services/accesscontrol"
+
+var builtInRoles = []accesscontrol.RoleDTO{
+	{
+		Name:    "grafana:builtin:users:read:self",
+		Version: 1,
+		Permissions: []accesscontrol.Permission{
+			{
+				Permission: "users:read",
+				Scope:      "users:self",
+			},
+			{
+				Permission: "users.tokens:list",
+				Scope:      "users:self",
+			},
+			{
+				Permission: "users.teams:read",
+				Scope:      "users:self",
+			},
+		},
+	},
+	{
+		Name:    "roles:adminUsers:viewer",
```

Comment:
After adding these predefined roles, I am really unsure about the difference between built in roles and predefines roles. Aren't these kind-of the same logical concept?